### ISSUE TITLE
v0.0.5 compatibility

### ISSUE THREAD

**Issue by JasonYao** [JasonYao/dotfiles#8]
## (1)
```
2018-11-06 06:19:55 PM [INFO::pydotfiles.models]: Action: Successfully completed action [Action=SYMLINK /Users/jason/.dotfiles/symlinks/env.symlink -> /Users/jason/.env]
2018-11-06 06:19:55 PM [INFO::pydotfiles.models]: Action: Starting action [Action=SYMLINK /Users/jason/.dotfiles/symlinks/utilities.symlink -> /Users/jason/.utilities]
2018-11-06 06:19:55 PM [INFO::pydotfiles.models]: Action: Successfully completed action [Action=SYMLINK /Users/jason/.dotfiles/symlinks/utilities.symlink -> /Users/jason/.utilities]
2018-11-06 06:19:55 PM [INFO::pydotfiles.models]: Install: Successfully installed module [name=symlinks]
2018-11-06 06:19:55 PM [INFO::pydotfiles.models]: Install: Installing module [name=macos]
  [ OK ] Keyboard: Press & hold is now disabled
  [ OK ] Keyboard: 0-delay key repeat is now enabled
  [ OK ] Keyboard: 0-delay initial key repeat is now enabled
  [ OK ] Finder: Show hard drives on desktop is now enabled
  [ OK ] Finder: Show external hard drives on desktop is already enabled
  [ OK ] Finder: Show removable media on desktop is already been enabled
  [ OK ] Battery: Show percent battery is now enabled
  [ .. ] Battery: Disabling computer/display sleep
Password:
setcomputersleep: Never
warning: this combination of display sleep and system sleep may prevent system sleep.
setdisplaysleep: Never
  [ OK ] Battery: Successfully disabled computer/display sleep
  [ OK ] Trackpad: Tracking speed is now set to max
  [ OK ] Trackpad: Natural scrolling is now disabled
  [ OK ] Trackpad: Show app expose gesture is now enabled
  [ OK ] iCloud: Default save location is now set to the local drive
  [ OK ] iCloud: .DS_Store file writing is now disabled on network drives
  [ .. ] Dock: Recent applications stack is not installed, installing now
  [ OK ] Dock: Recent applications stack is now installed
  [ .. ] Dock: Autohiding is not enabled, enabling now
  [ OK ] Dock: Autohiding is now enabled
  [ .. ] Dock: Magnification is not enabled, enabling now
  [ OK ] Dock: Magnification is now enabled
  [ .. ] Dock: Tilesize is not correctly set
  [ OK ] Dock: Tilesize is now correctly set
  [ .. ] Dock: Magnification tilesize is not correctly set
  [ OK ] Dock: Magnification tilesize is now correctly set
  [ OK ] Dock: Recent applications stack is already installed
  [ .. ] Dock: Contains non-default applications, killing off now
  [ OK ] Dock: All non-default applications are now sanitised
  [ .. ] Dock: Checking for all default app existences
2018-11-06 18:20:07.520 defaults[27982:75822] 
The domain/default pair of (com.apple.Dock, persistent-apps) does not exist
  [ .. ] Dock: Launchpad is not set on the dock, setting now
  [ OK ] Dock: Launchpad is now set on the dock
  [ .. ] Dock: Notes is not set on the dock, setting now
  [ OK ] Dock: Notes is now set on the dock
  [ .. ] Dock: iTunes is not set on the dock, setting now
  [ OK ] Dock: iTunes is now set on the dock
  [ .. ] Dock: App Store is not set on the dock, setting now
  [ OK ] Dock: App Store is now set on the dock
  [ .. ] Dock: System Preferences is not set on the dock, setting now
  [ OK ] Dock: System Preferences is now set on the dock
  [ .. ] Dock: Firefox is not set on the dock, setting now
  [ OK ] Dock: Firefox is now set on the dock
  [ OK ] Dock: All default apps are in place
  [ .. ] Dock: Application folder not found, adding now
  [ .. ] Dock: Applications folder is not set on the dock, setting now
  [ OK ] Dock: Applications folder is now set on the dock
  [ OK ] Bluetooth: BT is now disabled
No matching processes belonging to you were found
/System/Library/LaunchDaemons/com.apple.blued.plist: No such file or directory
/System/Library/LaunchDaemons/com.apple.blued.plist: No such file or directory
  [WARN] User: Guest account disabling is currently disabled for Sierra+ until bugfixed
2018-11-06 18:20:07.913 defaults[28023:75941] 
The domain/default pair of (com.apple.HIToolbox.plist, AppleEnabledInputSources) does not exist
  [ .. ] Languages: Adding Mandarin Pinyin input
  [ OK ] Languages: Traditional Chinese input via pinyin is now installed
  [ .. ] Languages: Traditional Chinese input via hand writing is not installed, installing now
  [ OK ] Languages: Traditional Chinese input via hand writing is now installed
  [ .. ] Languages: Adding US keyboard input
  [ OK ] Languages: Traditional Chinese input via pinyin is now installed
  [ .. ] Languages: Text input switcher is not enabled, enabling now
  [ OK ] Languages: Text input switcher is now enabled
  [WARN] Theme: Dark mode is now enabled, but requires a re-login for the menubar to reset
No matching processes belonging to you were found
  [ OK ] Spotlight: External disk indexing is now disabled
2018-11-06 18:20:08.303 defaults[28056:76030] 
The domain/default pair of (/Volumes/MacOS/.Spotlight-V100/VolumeConfiguration.plist, Exclusions) does not exist
2018-11-06 18:20:08.344 defaults[28058:76036] Could not write domain /Volumes/MacOS/.Spotlight-V100/VolumeConfiguration.plist; exiting
  [ OK ] Spotlight: Bootcamp partition has been excluded from normal indexing
  [ OK ] Mission Control: Automatic rearranging spaces is now disabled
  [ .. ] DNS: Host's DNS servers do not point at Google, setting DNS servers now
  [ OK ] DNS: Successfully set this host's DNS to point to google
  [ .. ] Network Search: Network search is enabled and points to a dangerous 3rd party, resetting to secure local state
  [ OK ] Network Search: Successfully disabled network search
  [ OK ] Firewall: Successfully set default to deny incoming traffic
  [ OK ] Firewall: Stealth mode has already been set
  [ OK ] Firewall: Traffic logging has already been set
  [ OK ] Firewall: Firewall has already been set
  [ OK ] Firewall: Allowing signed binaries has already been set
  [ OK ] OSX: Sane defaults installed
  [ .. ] Background Image: Checking background image status
  [ OK ] Background: Desktop image is now correctly set
2018-11-06 06:20:13 PM [INFO::pydotfiles.models]: Package: Attempting to install `coreutils`
2018-11-06 06:20:18 PM [INFO::pydotfiles.models]: Package: Successfully installed [Package=coreutils]
2018-11-06 06:20:18 PM [INFO::pydotfiles.models]: Caching: Dumping found packages into pydotfiles cache
2018-11-06 06:20:19 PM [INFO::pydotfiles.models]: Package: Attempting to install `wget`

```

**Comment by JasonYao** [comment]
Using https://ss64.com/ is a good place to start for those commands, especially if I'm going to start tagging this repo (e.g. 10.14-mojave)

**Comment by JasonYao** [comment]
This has been addressed as of #22
